src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        int record;
+        for (int i = 0; i < records.length; i++) {
+            int index = records[i].indexOf(':');
+            record = (int) Integer.parseInt(records[i].substring(index + 1));
```

Comment:
let's split data by ":" and save the result to the array and then use data from this array 
your solution will look more easy